dockstore-client/src/main/java/io/dockstore/client/cli/ArgumentUtility.java

```diff
@@ -57,11 +58,19 @@ public static void out(String arg) {
     }
 
     public static void err(String arg) {
-        LOG.error(arg);
+        if (LOG.isInfoEnabled()) {
+            LOG.error(arg);
+        } else {
+            System.err.println(ERROR_MESSAGE_START + arg);
+        }
     }
 
     private static void errFormatted(String format, Object... args) {
-        LOG.error((String.format(format, args)));
+        if (LOG.isInfoEnabled()) {
+            LOG.error(String.format(format, args));
```

Comment:
We might be able to eliminate some duplicate code by replacing this `if` with a call to `err(String.format(format, args))`

Reply:
Thanks, I have now done this.